draft-ietf-acme-acme.md

```diff
@@ -443,22 +443,19 @@ MUST authenticate the sender and verify any access control rules.
 Otherwise, the server MUST treat this request as having the same
 semantics as a GET request for the same resource.
 
-The server MUST allow GET requests for the following resources(see
-{{resources}}, in addition to POST-as-GET requests for these
-resources:
-
-* The directory resource
-* The newNonce resource
-* Certificate resources
-
-Allowing GET requests for the directory and newNonce resourcse
-enables clients to bootstrap into the ACME authentication system.
-GET requests for certificate resources allow non-ACME entities to
-consume certificates resulting from ACME interactions.  If a server
-regards certificate resources as sensitive, then it SHOULD assign
-them capability URLs {{?W3C.WD-capability-urls-20140218}} so that
-they cannot be guessed and thus have to be obtained from an
-authorized party.
+The server MUST allow GET requests for the directory and newNonce
+resources (see {{resources}}), in addition to POST-as-GET requests
+for these resources.  This enables clients to bootstrap into the
+ACME authentication system.
+
+The server MAY allow GET requests for certificate resources, in
+order to allow certificates to be fetched by a lower-privileged
+process, e.g., the web server that will use the referenced
+certificate chain.  (See {{?I-D.ietf-acme-star}} for more advanced
+cases.)  A server that allows GET requests for certificate resources
+can still provide them a degree of access control by assigning them
+capability URLs them capability URLs
```

Comment:
"them capability URLs" is there twice.

Reply:
Also please add a reference for "capability URLs", https://www.w3.org/TR/capability-urls/

Reply:
Suggest to add: The server MAY choose to allow GET requests to certain certificate resources but not to others. The server can base this decision on out-of-band knowledge (e.g., to allow GET requests to certificates owned by a certain account) or on order-specific information, such as the extension defined in {{?I-D.ietf-acme-star}}.